conf/solr/8.8.1/schema.xml

```diff
@@ -228,7 +228,166 @@
     <field name="filePersistentId" type="text_en" multiValued="false" stored="true" indexed="true"/>
     
     <!-- Dynamic Dataverse fields from http://localhost:8080/api/admin/index/solr/schema -->
```

Comment:
Can we please create more visible Search & Replace Guards here? Someone might delete that by accident, then things break.

Same for the `<copyField>` part.

Reply:
Fine with me. I was using whatever already was there to make sure the script works with schema.xml files that are already out there. Since that that is no concern, I'll add better markers in the schema.xml.